types/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// TypeScript Version: 3.5
+
+import {Element, Properties, Node} from 'hast'
+
+/**
+ * DSL to create virtual hast trees for HTML or SVG
+ *
+ * @param selector Simple CSS selector
+ * @param properties Map of properties
+ * @param children (Lists of) child nodes
+ */
+declare function hastscript(
+  selector?: string,
+  properties?: Properties,
+  children?: string | Node | Array<string | Node>
+): Element
```

Comment:
The Travis CI/DTSLint error seems to indicate that this currently can't support `children` as the second param, this may need to be overloaded

```suggestion
/**
 * DSL to create virtual hast trees for HTML or SVG
 *
 * @param selector Simple CSS selector
 * @param properties Map of properties
 * @param children (Lists of) child nodes
 */
 declare function hastscript(
  selector?: string,
  children?: string | Node | Array<string | Node>
): Element
declare function hastscript(
  selector?: string,
  properties?: Properties,
  children?: string | Node | Array<string | Node>
): Element
```

Reply:
That's my bad for hitting the merge button without actually testing it 😅